input/examples/organization-mitchells-hill-audiology.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Organization xmlns="http://hl7.org/fhir">
+    <id value="mitchells-hill-audiology"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-organization"/>
+    </meta>
+    <identifier>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="XX"/>
+            </coding>
+            <text value="ABN"/>
+        </type>
+        <system value="http://hl7.org.au/id/abn"/>
+        <value value="81129842694"/>
+    </identifier>
+  <type>
+    <coding>
+      <system value="http://www.abs.gov.au/ausstats/abs@.nsf/mf/1292.0"/>
+      <code value="8539"/>
+      <display value="Other Allied Health Services"/>
+    </coding>
+    <coding>
+      <system value="http://www.abs.gov.au/ausstats/abs@.nsf/mf/1292.0"/>
+      <code value="8539-3"/>
+      <display value="Audiology Service"/>
+    </coding>
+  </type>
+    <name value="Mitchells Hill Audiology"/>
+    <telecom>
+        <system value="phone"/>
+        <value value="(03) 5550 3095"/>
+    </telecom>
+    <telecom>
+        <system value="email"/>
+        <value value="reception@mitchellshillaudiology.example.com.au"/>
+        <use value="work"/>
+    </telecom>
+    <address>
```

Comment:
Add address type - this would match with the corresponding Location

Reply:
still missing address type

Reply:
done

Reply:
Done